src/web/core/src/test/java/org/geoserver/web/wicket/EnvelopePanelBoundingBoxTest.java

```diff
@@ -0,0 +1,109 @@
+/* (c) 2023 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.web.wicket;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Locale;
+import org.apache.wicket.Component;
+import org.apache.wicket.Session;
+import org.apache.wicket.markup.html.form.Form;
+import org.apache.wicket.util.tester.FormTester;
+import org.geoserver.web.FormTestPage;
+import org.geotools.geometry.jts.ReferencedEnvelope;
+import org.geotools.geometry.jts.ReferencedEnvelope3D;
+import org.geotools.referencing.CRS;
+import org.junit.Test;
+import org.opengis.referencing.FactoryException;
+
+public class EnvelopePanelBoundingBoxTest extends EnvelopePanelTest {
```

Comment:
Please don't extend an existing test class, the subclass will just end up running the tests of the base class one more time:

![image](https://user-images.githubusercontent.com/325433/221580933-af855734-6f9c-4dcc-a20c-a0d6d8cecd78.png)

Reply:
Switched parent to GeoServerWicketTestSupport